mrp_production_request/migrations/10.0.1.0.0/pre-migration.py

```diff
@@ -6,7 +6,7 @@
 
 field_renames = [
     ('mrp.production.request', 'mrp_production_request', 'product_uom',
-     'product_uom_id'),
+     'product_uom_id', 'date_planned', 'date_planned_start'),
```

Comment:
Then, this should be in a separate migration folder for version `10.0.1.1.0` (as migration need to happen for people that has already the module installed in v10 and is just upgrading).